_scicomputing/hutchgo_overview.md

```diff
@@ -0,0 +1,82 @@
+
+HutchGO is available for the advancement of the Hutch's mission.  There is no charge for Globus services.
+
+## Key Concepts
+
+### The Globus Identity
+
+As part of its services, Globus operates an _identity_ service.  Each individual who uses Globus has a Globus ID.  A Globus ID can be accessed by authenticating using a number of different sources- an institutional login (e.g. our Hutch O365 login), ORCID, Google account, or GitHub login can all be used to authenticate to your Globus ID.  You can have multiple login sources connected to the same Globus ID.
+
+> NOTE: while a single Globus ID can be linked to many different authentication sources, most of the operations described here require you to log into Globus with your HutchNet ID.  Logging into Globus with your ORCID will not get you access to the same resources as logging in with your "@fredhutch.org" institutional account.
+
+### Collections 
+
+Central to the use of Globus is the _collection_- a collection of paths to data on the underlying storage gateway. Data is transferred between collections.  Policies (like who can access data via Globus, how paths are shared, etc) are set on the collection.
+
+There are two kinds of collections- _mapped_ and _guest_.  A mapped collection requires that you have a HutchNet ID and that your HutchNet ID has access to the storage systems underneath Globus.  Your login to Globus is _mapped_ to a HutchNet ID on the storage system.
+
+> For example, if I have logged into Globus using my "mrg@fredhutch.org" institutional account and I access a collection that provides access to Fast file, I will only have the access privileges that my user account has on Fast.  In the mapped collection, Globus provides no "extra" privileges.
+
+In addition to mapped collections we have _guest_ collections.  Guest does not mean anonymous access- a guest collection allows external Globus users to access data _as you_ on the storage configured in the guest collection.  For that external user to access the data shared in a guest collection they must have a Globus ID, the guest collection must allow that ID to access the data, and your HutchNetID must have access to the storage on the back-end.
+
+> It is possible to enable public access to a collection, but "public" only means anyone with a Globus ID.  A Globus login is still required to access the collection's data
+
+### Important Roles
+
+Because Globus allows easy sharing of data and because we want to have the individual labs to have as much control as possible over how data is shared,  we are creating roles that control access to the various Globus capabilities.  Taking on a role means having a basic level of knowledge about how Globus works, constraints and controls necessary for the data being shared, and accepting responsibility for ongoing maintenance of data sharing within Globus.
```

Comment:
Specialized roles instead of Important ?

Then "we are creating specialized roles that grant access to additional Globas capabilities."

Reply:
Yes- since these are roles that are specific to our implementation I like the "specialized" term.  I've updated the title: "Roles for HutchGO Users" and used the "specialized" in the description.  

Updated in efa72bf.